package.json

```diff
@@ -7,23 +7,26 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "src/index.js",
+  "main": "ui-kit.cjs.js",
+  "module": "ui-kit.esm.js",
   "scripts": {
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "precommit": "lint-staged",
     "prepublish": "node ./scripts/generate-icon-exports.js",
     "prebuild": "node ./scripts/generate-base-colors.js && node ./scripts/generate-base-shadows.js && node ./scripts/generate-colors-for-story.js",
-    "prestart": "node ./scripts/generate-base-colors.js && node ./scripts/generate-base-shadows.js && node ./scripts/generate-colors-for-story.js",
-    "build": "build-storybook -o .public",
-    "start": "start-storybook -p 9002",
+    "prestorybook:start": "node ./scripts/generate-base-colors.js && node ./scripts/generate-base-shadows.js && node ./scripts/generate-colors-for-story.js",
+    "storybook:build": "build-storybook -o .public",
+    "build": "cross-env NODE_ENV=production rollup -c",
+    "start": "yarn storybook:start",
+    "storybook:start": "start-storybook -p 9002",
     "i18n:build": "mc-scripts extract-intl --build-translations --output-path=$(pwd)/i18n 'src/components/**/!(*.spec).js'",
     "lint": "jest --projects jest.eslint.config.js jest.stylelint.config.js",
     "lint:js": "jest --config jest.eslint.config.js",
     "lint:css": "jest --config jest.stylelint.config.js",
     "format": "npm run format:js && npm run format:css & npm run format:md",
-    "format:js": "prettier --write '{src,scripts,examples,.storybook}/**/*.js'",
-    "format:css": "prettier --parser postcss --write '{src}/**/*.css'",
-    "format:md": "prettier --parser markdown --write '{.github,src,examples}/**/*.md'",
+    "format:js": "prettier --write '{.storybook,examples,scripts,src}/**/*.js' *.js",
+    "format:css": "prettier --write --parser css '{.storybook,examples,src}/**/*.css'",
```

Comment:
PS: apparently `postcss` has been deprecated, it's only `css` now.

<img width="709" alt="image" src="https://user-images.githubusercontent.com/1110551/45231777-825d6a00-b2cd-11e8-871b-886da94fb018.png">